### PR TITLE
repren: add page

### PR DESCRIPTION
**Pull request by sbrl**
----

- [x] The page (if new), does not already exist in the repo.

- [x] The page (if new), has been added to the correct platform folder:  
      `common/` if it's common to all platforms, `linux/` if it's Linux-specific, and so on.

- [x] The page has 8 or fewer examples.

- [x] The PR is appropriately titled:  
      `<command name>: add page` for new pages, or `<command name>: <description of changes>` for pages being edited.

- [x] The page follows the [contributing](https://github.com/tldr-pages/tldr/blob/master/CONTRIBUTING.md) guidelines.

-----

https://github.com/jlevy/repren

### CODE REVIEW

File: pages/common/repren.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# repren
+
+> Multi-pattern string replacement and file renaming tool.
+
+- Do a dry-run renaming a folder of pngs with a literal string replacement:
+
+`repren --dry-run --literal --from '{{find_string}}' --to '{{replacement_string}}' *.png`
```
Comment (agnivade) [inline pages/common/repren.md]: `--rename` missing ?

File: pages/common/repren.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+- Do a dry-run renaming a folder of jpegs with a regular expression:
+
+`repren --rename --dry-run --from '{{regular_expression}}' --to '{{replacement_string}}' *.jpg *.jpeg`
+
+- Do a find-and-replace on the contents of a folder of csv files:
+
+`repren --from '{{([0-9]+) example_string}}' --to '{{replacement_string \1}}'`
```
Comment (agnivade) [inline pages/common/repren.md]: Shouldn't there be a ".csv" extension at the end ?

Comment (sbrl) [reply pages/common/repren.md]: Ah, yeah! I must have forgotten :P

File: pages/common/repren.md (hunk 1 above)
Comment (sbrl) [reply pages/common/repren.md]: yep! :P

**Comment by sbrl** [comment]
Oops! I did forget those tokens, didn't I :P

Thanks :D

**Pull request merged**